recipes/thorvg/all/conanfile.py

```diff
@@ -20,7 +20,7 @@ class ThorvgConan(ConanFile):
     description = "ThorVG is a platform-independent portable library that allows for drawing vector-based scenes and animations."
     license = "MIT"
     url = "https://github.com/conan-io/conan-center-index"
-    homepage = "https://github.com/thorvg/thorvg"
+    homepage = "https://www.thorvg.org/"
```

Comment:
By having the homepage be the git(hub) repository, automated tools can be used to check for new releases/information about the PRs/recipe in a easier way, I'd recommend keeping it to point to the repositories where possible. 

Wdyt @toge?

See for example https://github.com/conan-io/conan-center-index/issues/24327 which would have been pretty hard to generate without it

Reply:
@RubenRBS 
I agree with you.
I think it is very important to be able to automatically detect the version upgrades of a huge number of recipes.


By the way, as a reverse of this PR process, is it necessary to change recipes whose homepage is not github?